### ISSUE TITLE
Vertical Navigation Tree component breaking on Next.js

### ISSUE THREAD

**Issue by urkopineda** [lumada-design/hv-uikit-react#3783]
### Which version of NEXT UI Kit are you opening this bug for?

v5.x

### Latest version

- [X] I have tested the latest version

### No Hitachi confidential content

- [X] I'm not including any Hitachi confidential content

### Current behavior 😯

Using `@hitachivantara/uikit-react-core` version `5.36.0` with Next.js v13 is breaking when using the `HvVerticalNavigationTree` component. Last version `5.35.0` works just fine. The error shown is the next:

```
 ⨯ /workspace/node_modules/@mui/x-tree-view/internals/TreeViewProvider/useTreeViewContext.js:1
import * as React from 'react';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1153:20)
    at Module._compile (node:internal/modules/cjs/loader:1205:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at mod.require (/workspace/node_modules/next/dist/server/require-hook.js:64:28)
    at require (node:internal/modules/helpers:130:18)
    at Module.<anonymous> (/workspace/node_modules/@hitachivantara/uikit-react-core/dist/cjs/components/TreeView/internals.cjs:3:28)
    at Module._compile (node:internal/modules/cjs/loader:1241:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
```

### Expected behavior 🤔

`HvVerticalNavigationTree` component should just work.

### Steps to reproduce 🕹

CodeSandbox link: https://codesandbox.io/p/sandbox/eager-knuth-wh7smy

**Comment by urkopineda** [comment]
We are also using `@hitachivantara/uikit-react-lab` alongside `@hitachivantara/uikit-react-core`, and `@hitachivantara/uikit-react-lab` version `v.5.13.1` breaks our build. Switching to `v.5.13.0` makes it work again... maybe one dependency of both packages got updated and introduced this bug on Next.js? 🤔

Freezing `core` at `v.5.35.0` and `lab` at `v5.13.0` makes or build work again.

**Comment by plagoa** [comment]
Thanks for the heads up @urkopineda we'll look into it.

**Comment by MEsteves22** [comment]
Hello @urkopineda 👋

This was fixed on [@hitachivantara/uikit-react-lab@5.15.0](https://github.com/lumada-design/hv-uikit-react/releases/tag/%40hitachivantara%2Fuikit-react-lab%405.15.0) and [@hitachivantara/uikit-react-core@5.36.3](https://github.com/lumada-design/hv-uikit-react/releases/tag/%40hitachivantara%2Fuikit-react-core%405.36.3). Thanks for opening this issue.
